tox.ini

```diff
@@ -1,11 +1,6 @@
 [tox]
-envlist = python2.7,
-    python 3.3,	
-    python3.4, 	
-    python3.5,	
-    python3.6,	
-    flake8
-
+envlist =python3.6.0,
```

Comment:
@blac-siren . You could also put back the python versions

Reply:
@kzyGit I'll implement that.